bloom-instance/dns/alias/inputs.tf

```diff
@@ -0,0 +1,22 @@
+
+variable "zones" {
+  type        = map(string)
+  description = "A map of zone names to IDs"
+}
+
+variable "record" {
+  type = object({
+    # The DNS name for the record, ie google.com
+    name = string
+
+    # The type of record to create (CNAME, A, TXT, etc)
```

Comment:
is this always an A alias AWS resource record or are we making CNAMEs here as well, and simply calling the module 'alias'?

Reply:
There are two modules now, one that makes any type of record (which was used for `CNAME`s previously) and another that makes aliases (the new one introduced here).  They work a bit differently, both in implementation and under the hood on the AWS side.  It is technically possible to create types of alias records other than just `A`, but the `A` records are all we care about right now.  In the future there may be a need to add `AAAA` or `CNAME` alias records, although the `CNAME` aliases will still have the same problem that they can't be applied to the zone apex.  Regardless of the record type, they will all still be aliases, so a `CNAME` created with the `record` module will be a normal `CNAME` DNS record and a `CNAME` created with the `alias` module will be a `CNAME` alias instead.